ote/src/clj/ote/netex/netex.clj

```diff
@@ -0,0 +1,195 @@
+(ns ote.netex.netex
+  "Scheduled tasks to update gtfs file to s3 and later to database."
+  (:require
+    [taoensso.timbre :as log]
+    [clojure.java.io :as io]
+    [cheshire.core :as cheshire]
+    [ote.db.netex :as netex]
+    [specql.core :as specql]
+    [clojure.java.shell :refer [sh with-sh-dir]]
+    [clojure.string :as str]
+    [amazonica.aws.s3 :as s3]
+    [ote.config.netex-config :as config-nt]))
+
+(defn- path-allowed?
+  "Checks if path is in a system directory or similar not allowed place. Returns true if allowed, false if not."
+  [^String path]
+  (str/starts-with?
+    (.getAbsolutePath (clojure.java.io/file path))          ; TODO: should use .getCanonicalPath but osx
+    "/tmp/"))
+
+(defn delete-files-recursively! [f1]
+  (when (.isDirectory (io/file f1))
+    (doseq [f2 (.listFiles (io/file f1))]
+      (delete-files-recursively! f2)))
+  (io/delete-file f1 true))
+
+(defn cleanup-dir-recursive!
+  ;; TODO: replace using fileutils interop etc?
+  "Takes an path argument and does a simple sanity check before removing it and child directories recursively"
+  [path]
+  (if (path-allowed? path)
+    (delete-files-recursively! path)
+    (log/warn "Directory cleanup skipped, bad path = " path)))
+
+(defn- compose-chouette-import-gtfs-json [operator-name]
+  (cheshire/generate-string {:gtfs-import
+                             {:user_name "username-1"
```

Comment:
Could this be service name? I'm sure that we won't and transport operator users here, but "username-1" is not optimal